### PR TITLE
Final polishing

### LINKED ISSUE #118: Mutect2 process skipped
Hello,

I'm trying to run this pipeline with as many annotators as possible. However I can only output a couple of them. One that I cannot output is Mutect2 - it seems sarek skips that process, even if I explicit my germlineResource and pon files (including both indexes). What am I missing?

Code line:
`nextflow run nf-core/sarek --input FASTQ_SUB --tools Mutect2 --germlineResource somatic-b37_af-only-gnomad.raw.sites.vcf --germlineResourceIndex somatic-b37_af-only-gnomad.raw.sites.vcf.idx --pon somatic-b37_Mutect2-WGS-panel-b37.vcf.gz --pon_index somatic-b37_Mutect2-WGS-panel-b37.vcf.idx -profile singularity -c sarek.config
`

Thank you very much,
Pedro R.

### PR DESCRIPTION
**Pull request by MaxUlysse**
# nf-core/sarek pull request

- Final polishing
- ~~Trying to fix #118~~

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by MaxUlysse** [comment]
Couldn't reproduce #118 so I'm currently giving up on trying to fix it, and going on with the release.

**Comment by ggabernet** [comment]
Looks good to me, I just found two small things in the comments where I was not sure if it was intended or not

File: conf/test_split_fastq.config [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * -------------------------------------------------
  * Defines bundled input files and everything required
  * to run a fast and simple test. Use as follows:
- *   nextflow run nf-core/sarek -profile test
+ *   nextflow run nf-core/sarek -profile test_no_split_fastq
```
[the remark this answers is not in the document]
Comment (MaxUlysse) [reply conf/test_split_fastq.config]: Good catch, bad ctrl + C / ctrl + V

File: conf/test_targeted.config [hunk 2]
```diff
@@ -4,12 +4,12 @@
  * -------------------------------------------------
  * Defines bundled input files and everything required
  * to run a fast and simple test. Use as follows:
- *   nextflow run nf-core/sarek -profile test_targeted
+ *   nextflow run nf-core/sarek -profile test_no_split_fastq
```
[the remark this answers is not in the document]
Comment (MaxUlysse) [reply conf/test_targeted.config]: This one was actually good

[the remark this answers is not in the document]
Comment (MaxUlysse) [reply conf/test_targeted.config]: oh wait, no you're right

Comment (MaxUlysse) [inline conf/test_targeted.config]:
```suggestion
 *   nextflow run nf-core/sarek -profile test_targeted
```

**Pull request merged**